util/check_characters.py

```diff
@@ -37,7 +40,14 @@ def main():
             try:
                 next(reader, None)  # skip the file header (i.e. "transcript")
                 for row in reader:
-                    all_text |= set(str(row[2]))
+                    if args.unicode_variants:
+                        unicode_transcript = unicodedata.normalize("NFKC", row[2])
+                        if row[2] != unicode_transcript:
+                            raise ValueError(row[2])
+                    all_text |= set(row[2])
+            except ValueError as e:
+                print("Your input file", in_file, "contains at least one transript with unicode chars on more than one code-point: '{}'".format(e))
+                sys.exit(-1)
```

Comment:
This error message doesn't tell the user what they should do: NFKC normalization on their data.